cluster/common.sh

```diff
@@ -211,7 +214,7 @@ function get-kubeconfig-user-basicauth() {
 #   KUBE_USER
 #   KUBE_PASSWORD
 function gen-kube-basicauth() {
-    KUBE_USER=admin
+    KUBE_USER="admin"
```

Comment:
we should prefer single quotes for literals

Reply:
Good lesson!